src/olympia/devhub/views.py

```diff
@@ -1397,7 +1395,7 @@ def wrapper(request, *args, **kw):
                         return redirect(_step_url(max_step), addon.slug)
                 elif step != max_step:
                     # We couldn't find a step, so we must be done.
-                    return redirect(_step_url(6), addon.slug)
+                    return redirect(_step_url(5), addon.slug)
```

Comment:
As above, can you change this to use an url name instead of a number ? Then remove the `_step_url` function entirely.